src/open_inwoner/components/templates/components/Form/ChoiceRadio.html

```diff
@@ -11,4 +12,5 @@
         {% if choice.0 in data or checked %}checked="checked"{% endif %}
     />
     <label class="radio__label" for="id_{{ name }}_{{ index }}">{{ choice.1 }}</label>
```

Comment:
I think nesting labels may confuse assistive technology, can we get rid of the inner label?